src/main/java/org/sunbird/workflow/config/Configuration.java

```diff
@@ -279,4 +282,12 @@ public String getProfileServiceConfigPath() {
     public void setProfileServiceConfigPath(String profileServiceConfigPath) {
         this.profileServiceConfigPath = profileServiceConfigPath;
     }
+
+    public String getSignUpSupportServiceConfigPath() {
+        return signUpSupportServiceConfigPath;
+    }
+
+    public void setSignUpSupportServiceConfigPath(String signUpSupportServiceConfigPath) {
```

Comment:
Check the proper argument name

Reply:
please suggest a name for the config Path name